sklearn/ensemble/_forest.py

```diff
@@ -2421,7 +2449,7 @@ def fit_transform(self, X, y=None, sample_weight=None):
         X_transformed : sparse matrix of shape (n_samples, n_out)
             Transformed dataset.
         """
-        X = check_array(X, accept_sparse=['csc'])
+        X = self._validate_data(X, accept_sparse=['csc'])
```

Comment:
Uhm what is the reason that the common test where not failing for this transformer since we did not introduce `_validate_data` before

Reply:
`test_check_n_features_in_after_fitting` is applied to all estimators except those from modules listed in `N_FEATURES_IN_AFTER_FIT_MODULES_TO_IGNORE`. Every module where we add `n_features_in_` has to be removed from that list. This is done in this PR form ensemble.

Or do you think of another test?

Reply:
I was thinking about `check_n_features_in(name, estimator_orig)` but I can check on the side.
Sorry we merged a new PR that added some conflicts in `test_bagging` again.